en/04_Changelogs/beta/5.0.0-beta1.md

```diff
@@ -11,7 +66,86 @@ We will only consider merging post-beta breaking API changes if they are necessa
+
+#### Other issues
+
+`tractorcow/silverstripe-fluent` is temporarily being release under `silverstripe/silverstripe-fluent`.
+
+## Upgrading your third party module
+
+One of the primary purposes of the beta testing phase is to give third party maintainers and developers an opportunity to upgrade their modules to be compatible with Silverstripe CMS 5.
+
+### Planning your upgrade
+
+You should decide if you intend to do a new major release of your module to add support for Silverstripe CMS 5.
+
+Most of key Silverstripe CMS APIs have not change between Silverstripe CMS 4 and Silverstripe CMS 5. Take time to read this changelog to identify changes that might be problematic for your project.
+
+Depending on how complex your module is, it may be completely practical to create a version that works with both Silverstripe CMS 4 and Silverstripe CMS 5. We do not necessarily recommend this approach. But you may wish to consider it to minimise your maintenance burden.
+
+If you decide that you want to maintain distinct Silverstripe CMS 4 and Silverstripe CMS 5 versions of your module, then you should fork your current development branch.
+
+[info]
+If the name of your branch is a number, composer will allow it to be installed as an unstable tag. e.g.: If your module has a `7` branch, composer will alias the `HEAD` of that branch as `7.x-dev`. This will satisfy a constraint like `^7`.
+[/info]
+
+### Stop using deprecated code
+
+A good way to identify which APIs need to be updated is to [enable deprecation warnings](/Upgrading/Deprecations) and run your module against the latest Silverstripe CMS 4 release. This will let you remove any references to deprecated APIs in the Silverstripe CMS 4 version of your module.
+
+If you intend to remove some APIs in the Silverstripe CMS 5 version of your module, consider [adding deprecation warnings](/Contributing/Release_Process#Deprecating-API) to the Silverstripe CMS 4 version. This will simplify the upgrade process for projects using your module.
+
+### Upgrade your constraints
+
+Upgrade your composer requirement constraints to allow the installation of Silverstripe CMS 5 module. If your module has dual support for Silverstripe CMS 4 and 5, you'll want to use constraint similar to:
+```json
+"require": {
+    "silverstripe/framework": "^4 || ^5",
+    "silverstripe/cms": "^4 || ^5",
+    "silverstripe/admin": "^1 || ^2"
+}
+```
+
+Otherwise, simply increment your existing constraints to target the Silverstripe CMS 5 version of our modules.
+
+### Ship a Silverstripe CMS 5 version of your module
+
+Obviously, you'll want to test your module against a Silverstripe CMS 5 installation to identify and fix any problems. Once you're confident that your module works well with Silverstripe CMS 5, you'll want to tag a new release.
```

Comment:
I don't know what value this paragraph adds - the first word in it makes it clear it's stuff people already know.

Reply:
It felt a bit weird to stop the explication at "update your constraints". I can drop the "Obviously" if it makes it sound condescending.